lib/take_order.rb

```diff
@@ -0,0 +1,41 @@
+require 'rubygems' # not necessary with ruby 1.9 but included for completeness 
+require 'twilio-ruby'
+require './lib/order.rb'
+
+class Checkout
+
+  def initialize(money, total)
+    @money = money
+    @total = total#doesn't access order total, have to insert total
+    #upon initialising checkout
+  end
+
```

Comment:
You could consider passing an order to the checkout, so that the checkout can access order total.